src/server/firefox/ffBrowser.ts

```diff
@@ -91,6 +91,7 @@ export class FFBrowser extends Browser {
 
   async _initVersion() {
     const result = await this._connection.send('Browser.getInfo');
+    this._isMac = result.userAgent.includes('Macintosh');
```

Comment:
If the user would overwrite the user agent when the host is e.g. Mac and the new User-Agent is Linux, this would not work or am I wrong?

Reply:
This userAgent is the real one that comes from the browser, not the one overwritten by the user. So it depends what you mean by not work".

Reply:
We don't emulate `navigator.platform`, but if we started it might make sense to wrap this behavior in there.